studio/components/interfaces/BillingV2/Usage/Infrastructure.tsx

```diff
@@ -118,10 +122,10 @@ const Infrastructure = ({
             <SectionContent section={attribute}>
               {attribute.key === 'disk_io_consumption' && (
                 <>
-                  {currentBillingCycleSelected && highestIoBudgetConsumption >= 100 ? (
-                    <Alert withIcon variant="danger" title="IO Budget for today has been used up">
+                  {latestIoBudgetConsumption >= 100 ? (
```

Comment:
If the user selects a custom timeframe (not the current billing cycle), we probably don't want to show that alert, that's why we only showed it if `currentBillingCycleSelected` is true

Reply:
Right, when the user selects previous billing cycle view, this warning shouldn't appear.

Ideally, the warning shouldn't disappear when user selects a different timeframe though. Will see how I can adapt 👍